htdocs/commande/class/api_orders.class.php

```diff
@@ -623,6 +623,12 @@ public function put($id, $request_data = null)
 			throw new RestException(400, 'Error while updating availability');
 		}
 
+        // Update demand reason
```

Comment:
Because demand_reason_id is a property of the order object, i think it should be better to edit the update() method of order to include  demand_reason_id. I don't understand why this was done in a different method.